Jenkinsfile

```diff
@@ -33,6 +35,7 @@ pipeline {
     stages {
         stage('Calculating build parameters'){
             steps {
+                dockerd {}
```

Comment:
❓ I'm guessing this has jenkins run the build in our docker container instead of creating it's own or something like that?